packages/console/src/consts/language.ts

```diff
@@ -0,0 +1,6 @@
+import { Language } from '@logto/phrases';
+
+export const languageOptions = [
+  { value: Language.English, title: 'English' },
+  { value: Language.Chinese, title: '中文' },
+];
```

Comment:
Nit: This may cause the community contributors hard to add their language packs for us.
Maybe we need an issue to discuss this.

Reply:
we can combine this into a central place. @demonzoo would you like to add an issue before merging? thanks

Reply:
ah

Reply:
actually just put em in phrases?

Reply:
ok, np

Reply:
Updated!